backend/space/models.py

```diff
@@ -194,6 +195,38 @@ class LandscapeFeature(Field, models.Model):
     """
 
     space = models.ForeignKey(
-        to=SpaceDescription, on_delete=models.CASCADE, related_name="landscape_features"
+        to=SpaceDescriptionBase,
+        on_delete=models.CASCADE,
+        related_name="landscape_features",
     )
     landscape = models.CharField(max_length=512, blank=False)
+
+
+class SpaceDescription(Historical, SpaceDescriptionBase, models.Model):
+    """
+    An aggregate model that represents a space in history. This model is based on one or multiple SourceDescriptions and other sources that are not part of this database.
+    """
+
+    pass
+
+
+class SpaceDescriptionDescription(
+    SourceDescription, SpaceDescriptionBase, models.Model
+):
+    """
+    A description of a space description in a source.
+    """
```

Comment:
"A description of a space description"?

This distinction is not really appropriate here; as this awkward docstring highlights, `SpaceDescription` was already a type of description.

In the case of spaces, as I understand it, source texts are so different in what they describe, and so vague, that it is a futile exercise to try to link specific places between sources. This is why we ended up with a model where a single `SpaceDescription` contains several _aspects_ that may have been described, each of which may link to a historical reference point, such as a kingdom or diocese.

So to map the `space` app to the `Historical`/`Description` division... When I wrote this originally, I imagined we would divide it like this:
- `PoliticalRegion`, `EcclesiasticalRegion`, etc., are all `Historical` models
- `SpaceDescription` is a `SourceDescription`